source/setup.py

```diff
@@ -46,43 +42,29 @@ def determine_dll_type(self, imagename):
 	return orig_determine_dll_type(self, imagename)
 DllFinder.determine_dll_type = determine_dll_type
 
-class py2exe(distutils_buildexe.py2exe):
-	"""Overridden py2exe command to:
-		* Add a command line option --enable-uiAccess to enable uiAccess for the main executable and EOA proxy
-		* Add a manifest to the executables
+
+def _parsePartialArguments() -> argparse.Namespace:
+	"""
+	Adds a command line option --enable-uiAccess to enable uiAccess for the main executable and EOA proxy
+	Allows py2exe to parse the rest of the arguments
 	"""
+	partialParser = argparse.ArgumentParser()
+	partialParser.add_argument(
+		"--enable-uiAccess",
+		dest="uiAccess",
+		action="store_true",
+		help="enable uiAccess for the main executable"
+	)
+	partialArgs, _argslist = partialParser.parse_known_args(("--enable-uiAccess", ))

```

Comment:
It looks like this is a left over from testing - I believe the intent is to parse arguments from `sys.argv` and not from the explicitly provided list.

Reply:
This is intentional - this allows us to partially parse the enable uiAccess flag, but allow py2exe to parse the rest of the parameters themselves

Reply:
This has no effect that you think it does. By passing `("--enable-uiAccess", )` to the `parse_known_args` you are using the provided list of arguments rather than `sys.argv`. As a result `uiAccess` is always set to `True`, and the value provided from the CLI is ignored. Besides py2exe does not do any command line handling with the new freeze API, and looking at the arguments we're passing to the setup script most of them should be removed:
- `"-vv"` should be replaced with the `verbose"` set to 2 in the `options` dictionary provided as a parameter to `py2exe.freeze
- `"build", "--build-base", buildDir.abspath,` is a left over from when py2exe relied on setuptools, and can safely be removed
- `--dist-dir` can no longer be specified from the command line natively, so we need to either hard-code it in the call to freeze as you did in this PR, or add it as a parameter to the CLI parser ourselves, and use the CLI argument in the call to freeze.

Reply:
Ah I understand what you mean now, you were suggesting

```suggestion
	partialArgs, _argslist = partialParser.parse_known_args(sys.argv)
```

Reply:
this should be fixed now